src/rust/sdk/example/src/main.rs

```diff
@@ -0,0 +1,353 @@
+    let module_id = ModuleId::from(
+        MoveModuleId::from_str(&format!("{}::faucet", faucet_address))
+            .map_err(|a| EconiaError::InvalidModuleId(a.to_string()))?,
+    );
+    let entry =
+        EntryFunction::new(
+            module_id.clone(),
+            ident_str!("mint").to_owned(),
+            vec![coin.clone().into()],
+            vec![
+                bcs::to_bytes(&amount)?,
+            ],
+        );
+    econia_client.submit_tx(entry).await?;
+    Ok(())
+}
+
+/// Returns (best bid level, best ask level)
+pub async fn get_best_levels(view_client: EconiaViewClient<'_>, market_id: u64) -> EconiaResult<(Option<PriceLevel>, Option<PriceLevel>)> {
+    let levels = view_client.get_price_levels_all(market_id).await?;
+
+    let best_bid_level = if !levels.bids.is_empty() {
+        Some(levels.bids.get(0).unwrap().clone())
+    } else { None };
+
+    let best_ask_level = if !levels.bids.is_empty() {
+        Some(levels.bids.get(0).unwrap().clone())
+    } else { None };
+
+    Ok((best_bid_level, best_ask_level))
+}
+
+pub async fn place_limit_orders_at_market(view_client: EconiaViewClient<'_>, econia_address: AccountAddress, e_apt: &TypeTag, e_usdc: &TypeTag, market_id: u64, size_lots_of_base: u64, min_bid_price_ticks_of_quote: u64, max_bid_price_ticks_of_quote: u64) -> EconiaResult<(EntryFunction, EntryFunction)> {
+    let (best_bid_level, best_ask_level) = get_best_levels(view_client, market_id).await?;
+
+    let bid_entry = if let Some(best_bid_level) = best_bid_level {
+        let best_bid_price = best_bid_level.price;
+        place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, size_lots_of_base, best_bid_price + 1, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?
+    } else {
+        place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, size_lots_of_base, min_bid_price_ticks_of_quote, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?
+    };
+
+    let ask_entry = if let Some(best_ask_level) = best_ask_level {
+        let best_ask_price = best_ask_level.price;
+        place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, size_lots_of_base, best_ask_price - 1, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?
+    } else {
+        place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, size_lots_of_base, max_bid_price_ticks_of_quote, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?
+    };
+
+    Ok((bid_entry, ask_entry))
+}
+
+pub async fn report_best_price_levels(view_client: EconiaViewClient<'_>, market_id: u64) -> EconiaResult<()> {
+
+    let (best_bid_level, best_ask_level) = get_best_levels(view_client, market_id).await?;
+
+    if best_bid_level.is_none() && best_ask_level.is_none() {
+        println!("There is no EAPT being bought or sold right now");
```

Comment:
When printing output to the user let's use a lowercase e for the prefix: eAPT and eUSDC